### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by rjocoleman** [patrickrills/Duffy#21]
Hey @patrickrills this is awesome.

Thanks so much for making it and open sourcing.

I haven’t dug in too much yet - I’m keen to kick in a PR if your open to this feature?

Goal streak tracking - on the iOS and watch app showing how many days goal has been met in a “streak”.

Potentially adding this into any notifications - “way to go 63 day streak” etc..

**Comment by patrickrills** [comment]
Yeah that sounds good. Something I've always thought about but didn't know how much interest there was.
